md5sums.txt

```diff
@@ -0,0 +1 @@
+eb1398aa6c27cec022a933a1502590fe assets/deep_histopath_model.hdf5
```

Comment:
Can you verify this is correct? It looks like this is causing Travis to fail. Otherwise LGTM